src/language-js/utils.js

```diff
@@ -885,6 +885,77 @@ function isLongCurriedCallExpression(path) {
   );
 }
 
+/**
+ * @param {import('estree').Node} node
+ * @param {number} depth
+ * @returns {boolean}
+ */
+function isSimpleCallArgument(node, depth) {
+  if (depth >= 2) {
+    return false;
+  }
+  const isChildSimple = child => isSimpleCallArgument(child, depth + 1);
+  if (
+    !node ||
+    node.type === "Literal" ||
+    node.type === "BooleanLiteral" ||
+    node.type === "NullLiteral" ||
+    node.type === "NumericLiteral" ||
+    node.type === "StringLiteral" ||
+    node.type === "Identifier" ||
+    node.type === "ThisExpression" ||
+    node.type === "Super" ||
+    node.type === "BigIntLiteral" ||
+    node.type === "PrivateName" ||
+    node.type === "ArgumentPlaceholder" ||
+    node.type === "RegExpLiteral"
```

Comment:
What about `Import`? The following case is edge case but it looks strange for me.
**Input**
```js
point().x(4).y(3).z(fooooo("foo")).plot();
point().x(4).y(3).z(import("foo")).plot();
```

**Output**
```js
point().x(4).y(3).z(fooooo("foo")).plot();
point()
  .x(4)
  .y(3)
  .z(import("foo"))
  .plot();
```

[Playground Link](https://deploy-preview-6685--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAHCBLWAKAlAOgA9sAWAgT2wGYCAvbAMwgmwB0QmJ3cDUAbCDDwBuVlHRYhBYmXyUa+ehgC26AE5D2nbrwFTRUEABoQEVDAzQAzslABDNWogB3AAoOENlHYBumACbGIABGanZgANZwMADKqOFYAObIMGoArnAmABYwynwA6lkY8FbxYHAxnsUYPsXkyOBWNiZYVnAarmGJynbIDHZ8bSYAVlaEAEJhkdExdspwADJYcH0DQyCjhDFJfHAAimmCK0j9g5loDm1qDcF2wXB8QahObflhqA3PcFc+KyYAjod4J0zF4QHYrABaKBwOD+OFBNRwQEYJGdOzdXonNbnNrKDApdK4nb7IHHU7rGB3fIYfwwLLIABMJlSdgwfCSAGEIMoeg0oNA-iA0m0ACp3LwU84+DIASSg8NgMTAagw5gAggqYjByLtVmcTM9JDS6QykAAOAAMAF9rUA)

Reply:
I'm really unsure about regexps. Things like
```js
base
    .replace(/^\w*:\/\//, '')
    .replace(/\/$/, '')
    .split('/').length
```
don't look good inline at all.